presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -80,6 +80,8 @@
     private DataSize s3MultipartMinFileSize = new DataSize(16, MEGABYTE);
     private DataSize s3MultipartMinPartSize = new DataSize(5, MEGABYTE);
 
+    private boolean parquetColumnIndexAccess = true;
```

Comment:
Sorry, I just looked at the Hive code again, and it looks like the default is to respect the column names in the Parquet file over the Hive metastore, so this should be false.